crates/holochain/src/core/ribosome/real_ribosome.rs

```diff
@@ -732,105 +785,88 @@ impl RealRibosome {
 
     pub fn do_wasm_call_for_module<I: Invocation>(
         &self,
-        call_context: CallContext,
         invocation: &I,
         zome: &Zome,
-        to_call: &FunctionName,
-        module: Arc<Module>,
-    ) -> Result<Option<ExternIO>, RibosomeError> {
-        if module.info().exports.contains_key(to_call.as_ref()) {
```

Comment:
This is checked in the calling function now, so that the module doesn't need to be passed in here.